signature/sigstore/rekor/rekor_test.go

```diff
@@ -0,0 +1,126 @@
+//go:build !containers_image_rekor_stub
```

Comment:
If we drop the dependency, we might not need the build tag either. (Besides, Debian now has Rekor packaged…)

@siretart WDYT about dropping this build tag and always including Rekor support? See the `go.mod` changes in this PR for the (non-)impacted dependencies.